pkgs/development/python-modules/discid/default.nix

```diff
@@ -0,0 +1,27 @@
+{
+  stdenv, lib, pkgs,
+  buildPythonPackage, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "discid";
+  version = "1.1.0";
+  name = "${pname}-${version}";
+
+  meta = {
+    description = "Python binding of libdiscid";
+    homepage    = "https://python-discid.readthedocs.org/";
+    license     = lib.licenses.lgpl3Plus;
+    platforms   = lib.platforms.linux;
+  };
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "b39d443051b26d0230be7a6c616243daae93337a8711dd5d4119bb6a0e516fa8";
+  };
+
+  patchPhase = ''
+    substituteInPlace discid/libdiscid.py \
+      --replace '_open_library(_LIB_NAME)' "_open_library('${pkgs.libdiscid}/lib/libdiscid.so.0')"
```

Comment:
It's funny, I remember I have try this and it failed so I have to put `pkgs` back in.
Let my try again.